.travis.yml

```diff
@@ -37,6 +39,9 @@ before_install:
 before_script:
   - bundle exec rake db:create db:migrate
 
-bundler_args: --without development --deployment --retry=3 --jobs=3
+install:
+  - bash -c "if [ '$RAILS42' == '1' ]; then bundle update --retry=3 --jobs=3 rails rails-observers; fi"
+  - bash -c "if [ '$RAILS_MASTER' == '1' ]; then bundle update --retry=3 --jobs=3 arel rails rails-observers seed-fu; fi"
```

Comment:
Theoretically that list should be only rails. Thats why I wanted to make the other dependencies work in multiple versions, so we would not need to have to update them on the lock file.